src/mirk_tableaus.jl

```diff
@@ -115,8 +115,8 @@ function constructMIRK6(S::BVPSystem)
 
     # Interpolant tableau
     s_star = 9
-    c_star = [7 // 16, 1 // 8, 9 // 16, 3 // 8]
-    v_star = [7 // 16, 1 // 8, 9 // 16, 3 // 8]
+    c_star = [7 // 16, 3 // 8, 9 // 16, 1 // 8]
+    v_star = [7 // 16, 3 // 8, 9 // 16, 1 // 8]
```

Comment:
Is a convergence test checking htis?

Reply:
The interpolation coefficients don't affect the convergence test, but they did affect the ensemble test. I found the previous interpolation coefficients in MIRK6 are wrong, so I directly changed them in this PR.

```fortran
c_star(1) = 7.0d0/16.0d0
c_star(2) = 3.0d0/8.0d0
c_star(3) = 9.0d0/16.0d0
c_star(4) = 1.0d0/8.0d0

v_star(1) = 7.0d0/16.0d0
v_star(2) = 3.0d0/8.0d0
v_star(3) = 9.0d0/16.0d0
v_star(4) = 1.0d0/8.0d0
```

Reply:
Let's do a follow up with some convergence or regression test on the interpolation. The interpolation coefficients should be used as the post solution interpolation, so I think that's the first priority: make `sol(t)` work at higher order using this continuous definition (and save the `k`'s to the solution). Once that's the case, we can add a dense convergence test which will use non-grid values to test the convergence, and that should then test the accuracy of the interpolation.